src/store/slices/currentPageSlice.ts

```diff
@@ -0,0 +1,22 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+import { IBook } from "../../interfaces";
+
+export interface CurrentPageItemsInitialState {
+  currentPageItems: IBook[];
+}
+const initialState: CurrentPageItemsInitialState = {
+  currentPageItems: [],
+};
+
+export const currentPageItemsSlice = createSlice({
+  name: "currentPageItems",
+  initialState: initialState,
```

Comment:
```suggestion
  initialState,
```
много где у тебя такое вижу. если название поля совпадает с названием переменной просто передавай таким образом. уверен, что ты в курсе, но почему-то не юзаешь